posthog/client.py

```diff
@@ -398,44 +414,39 @@ def feature_enabled(self, key, distinct_id, default=False, *, groups={}):
             for flag in self.feature_flags:
                 if flag["key"] == key:
                     feature_flag = flag
-                    if feature_flag.get("is_simple_flag"):
-                        rollout_percentage = (
-                            feature_flag.get("rollout_percentage")
-                            if feature_flag.get("rollout_percentage") is not None
-                            else 100
-                        )
-                        response = _hash(key, distinct_id) <= (rollout_percentage / 100)
-        if response == None:
+                    try:
+                        flag_filters = feature_flag.get("filters") or {}
+                        aggregation_group_type_index = flag_filters.get("aggregation_group_type_index")
+                        if aggregation_group_type_index is not None:
+                            focused_group_properties = group_properties[
```

Comment:
This is tricky. We should hash out how we expect users to add properties here.

I think what you're doing makes sense, specially when different groups can have the same `name` property. However at the same time, feature flags support a single group at a time, so we'd expect only properties from a single group per call..

This is more future proof, but also a bit annoying for current user needs.

Reply:
EIther works for me. When only one group is in focus, how does this function know which group is being used?

Reply:
`aggregation_group_type_index` ? Not sure I get the question

Reply:
Oh, so upon calling posthog.get_feature_flag() you're assumed to already know what group type the flag in question is, so you only pass in the group properties related to that group?

Reply:
Yeah pretty much, since they're the ones who defined the flag

Reply:
Ah, thought about this a bit more, and I think your format makes sense, since we'll want to send these properties to `/decide` as well, when we do fallback, so that these props can be overridden.

Reply:
And /decide evaluates multiple flags, and we don't know which of those have which groups